ReqnrollProject1/Pages/LSPage.cs

```diff
@@ -0,0 +1,153 @@
+﻿using mars.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V135.Audits;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mars.Pages
+{
+    public class LSPage : CommonDriver
+    {
+        public void languagePage(IWebDriver driver, String Languages, String Level)
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://medium.com/@jignect/mastering-selenium-c-with-nunit-in-depth-guide-to-page-object-model-pom-and-data-object-model-62a9c71aa01c

https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/

Reply:
and split pages into skills and language